DataFormats/Detectors/CTP/include/DataFormatsCTP/ctpRateFetcher.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef COMMON_CCDB_CTPRATEFETCHER_H_
+#define COMMON_CCDB_CTPRATEFETCHER_H_
+
+#include <string>
+
+#include "CCDB/BasicCCDBManager.h"
+#include "DataFormatsParameters/GRPLHCIFData.h"
+
+namespace o2
+{
+namespace ctp
+{
+class CTPRunScalers;
+class CTPConfiguration;
+
+class ctpRateFetcher
```

Comment:
Classes should start with an uppercase

Reply:
The idea was that is exactly same as in O2Physics:
https://github.com/AliceO2Group/O2Physics/blob/master/Common/CCDB/ctpRateFetcher.h
and later is then replaces by the one from O2.
OK, it can be wrapped or changed everywhere in O2Physics where used.
I can change it if you confirm.
Let us see if @mpuccio has any comments.

Reply:
I would not delay this PR for this. 
My advise is nevertheless to fix it soon. 
Coding guidelines: https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Type_Names#Type_Names